config/routes.rb

```diff
@@ -1,11 +1,42 @@
 Rails.application.routes.draw do
+
   devise_for :users
 
+  get "/admin", to: "admin#landing_page", as: "landing_page_admin"
+  get "/admin/volunteers", to: "admin#volunteer_admin", as: "volunteer_admin"
+  get "/yearbook", to: "admin#yearbook", as: "yearbook_admin"
+
+  get "/public", to: "public#landing_page", as: "landing_page_public"
+  get "/about", to: "public#about", as: "about_public"
+  get "/community_resources_list", to: "public#community_resources", as: "community_resources_public"
+  get "/news", to: "public#news_and_announcements", as: "news_and_announcements_public"
+  get "/share", to: "public#share", as: "share_public"
```

Comment:
I think this makes sense for now but i think there's some value to all admin functionality being under an `/admin` namespace (and possibly the corresponding controllers being namespaced that way as well, eg `Admin::AnnouncementsController`). I don't have a strong argument from a code standpoint, but i think there's a familiarity and confidence users feel when they see `/admin` in the url, you know? I think that change would probably go hand in hand with public endpoints getting their own minimal controllers.